extension/chrome/settings/index.ts

```diff
@@ -309,6 +311,38 @@ Catch.try(async () => {
     }));
   };
 
+  const renderBlogPostList = async () => {
+    const errorRetrievingPosts = () => {
+      Xss.sanitizeAppend('.blog_post_list', `<div class="line>Error retrieving blog posts...try again later.</div>\n`);
+    };
+    try {
+      const res = await Api.ajax({
+        url: 'https://flowcrypt.com/feed',
+        method: 'GET',
+        dataType: 'json',
+        async: true
+      }, Catch.stackTrace());
+
+      interface BlogPostData {
+        url: string;
+        title: string;
+        date: string;
+      }
+      if (res instanceof Array) {
```

Comment:
You don't need to test this, if there is error, it will be thrown and reported, so we can deal with it.